misc/docerina/src/main/java/org/ballerinalang/docgen/docs/utils/BallerinaDocUtils.java

```diff
@@ -58,6 +60,24 @@ public static void packageToZipFile(String sourceDirPath, String zipFilePath) th
         }
     }
 
+    public static byte[] checkHash(byte[] contentInBytes, String algorithm) {
+        MessageDigest digest;
+        try {
+            digest = MessageDigest.getInstance(algorithm);
+        } catch (NoSuchAlgorithmException e) {
+            throw new IllegalArgumentException(e);
+        }
+        return digest.digest(contentInBytes);
+    }
+
+    public static String bytesToHex(byte[] bytes) {
```

Comment:
Unless these methods are used outside the package, we can make them package private instead public

Reply:
It's used outside the package, so need to keep the access modifier as public.